### DISCUSSION TITLE
Tengo una versión de msys

### DISCUSSION THREAD

**Discussion by samusisto** [INGCOM-UNRN-P1/cursada-2023#11] [Preguntas y respuestas] [answered]
Hola. tengo una versión instalada previamente y cuanto le pido el comando «versión» me indica esto:
```
**_samus@LAPTOP-5P712O77_** _MSYS_ ~
**_$_** gcc ~~version
/usr/lib/gcc/x86_64-pc-msys/11.3.0/../../../../x86_64-pc-msys/bin/ld: cannot find ~~version: No such file or directory
collect2: error: ld returned 1 exit status
```
¿está bien? ¿está mal? ¿cómo saber?

nota:  la nueva versión la instalé en **_c:/_** donde ya tenía ´una´ pero no la puedo ejecutar ni con guía...

**Comment by martinvilu** [comment, marked as answer]
Estás usando `gcc ~~version` en lugar de `gcc --version`.

Esto no llama a la opción `version` en `gcc`, sino que es tomada como nombre de archivo

**Reply by samusisto** [reply]
Increible ! (?)
listo ya lo solucioné y lo commitee

**_samus@LAPTOP-5P712O77_**  _MSYS_ ~
**_$_** gcc --version
gcc (GCC) 11.3.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

«cerranding»
